actions/upload_pacbio_sequencerun.py

```
@@ -0,0 +1,65 @@
+        payload = {
+            "run_uuid": run_uuid,
+            "run_name": run_name,
+            "ccs_execution_mode": self._get_ccs_execution_mode(collections),
+            "requires_demultiplex": requires_demultiplex,
+            "collection_paths": collection_paths,
+            "collections": collections,
+        }
+        headers = {"Authorization": "Api-Key {}".format(processing_api_access_key)}
+
+        try:
+            response = requests.post(url, json=payload, headers=headers)
```

Ruff (0.16.1)

[error] Probable use of `requests` call without timeout

(S113)